submissions/adeyemiJuwon/.vscode/settings.json

```diff
@@ -1,3 +1,3 @@
 {
-    "liveServer.settings.port": 5503
+    "liveServer.settings.port": 5505
```

Comment:
What is this file for?

Reply:
I think my vscode editor create it.
Can i delete it

Reply:
Yes